pkg/api/const.go

```diff
@@ -453,6 +453,8 @@ const (
 	DefaultKubernetesMaxPodsAzureCNI = "30"
 	// DefaultKubernetesAPIServerEnableProfiling is the config that enables profiling via web interface host:port/debug/pprof/
 	DefaultKubernetesAPIServerEnableProfiling = "false"
+	// DefaultKubernetesAPIServerVerbosity is the default verbosity setting for the apiserver
+	DefaultKubernetesAPIServerVerbosity = "4"
```

Comment:
While we're making this improvement, let's go ahead and change the default to 2.